### DISCUSSION TITLE
New here and hoping someone can help

### DISCUSSION THREAD

**Discussion by KBNovice12** [UltimaHoarder/UltimaScraper#1750] [Q&A]
I don't have any coding experience but would like to try this scrapper but I don't know how to start. I'm sure noobs like me are annoying like trying to teach a toddler 2 + 2 but I'm hoping someone out there will help. Do I need to install some program to apply the code? I'm sure there's tons of requests but really hoping someone can show me the way. Thank you ladies and gentlemen in advance

**Comment by JaegerMedia** [comment]
1. Get Ubuntu through the Microsoft Store installed and updated.  Or run a Linux VM.  It's a lot easier to work with things like this IMO

2. Clone the repository with git clone https://github.com/DIGITALCRIMINAL/OnlyFans

3. Also install https://github.com/M-rcus/OnlyFans-Cookie-Helper into Firefox (Just click the .xpi asset in Firefox browser)

4. Make sure you have Python 3.9 and Pip3.  Install prerequisite packages with pip3 install -r requirements.txt inside of the OnlyFans folder you cloned.

5. Run python 3.9 start_ofd.py

6. Use the addon to get the cookie information, clear ~/OnlyFans/.profiles/OnlyFans/default/auth.json and paste in the info from the helper when prompted to fill this information in.

7. Run the script with python3.9 start_ofd.py from the repo folder

I don't know how familiar you are with Linux but with some googling this is achievable to a beginner.

**Reply by KBNovice12** [reply]
Wow thanks so much friend was really more of a shot in the dark but very kind of you to point me in the right direction! For now I think I will try with Ubuntu since I am familiar with micro store. I am not quite sure what you mean by clone the repository... do you mean to copy/paste the codes (i.e. after clicking "Consistent username"). Again thanks for your patience as this must be so simple for you. I also found a YT tutorial for python so I will try to help myself and maybe answer my question.
